espnet/nets/pytorch_backend/e2e_st_transformer.py

```diff
@@ -0,0 +1,466 @@
+# Copyright 2019 Shigeki Karita
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
Maybe you can add your name.
Also, could you add some comments which part mainly you modified/added from the original ASR transformer code (of course translation method, but are there anything else)?

Reply:
ASR/MT multi-task only.